alibi_detect/utils/keops/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .kernels import GaussianRBF
```

Comment:
Should this be protected with `import_optional`? @mauicv

Reply:
ah! yeah, perhaps it should! Is it part of the public api?

Reply:
just for reference, this has been protected as of [#71142d7](https://github.com/SeldonIO/alibi-detect/pull/548/commits/71142d79b88c1b98085c86fbf5bf8e027fd6870f)